game.js

```
@@ -1600,12 +1680,15 @@ function drawDebugPanel() {
 }
 
 // Add debug panel to UI drawing in drawPlayers function
-function drawPlayers() {
+function drawPlayers(interpolation = 1) {
```

Biome (1.9.4)

[error] Shouldn't redeclare 'drawPlayers'. Consider to delete it or rename it.

'drawPlayers' is defined here:


(lint/suspicious/noRedeclare)